src/lib/chimp-spec.js

```diff
@@ -0,0 +1,75 @@
+const Chimp = require('./chimp');
```

Comment:
Interestingly, I'm using the same pattern you're making the PR to enable you to do